src/image/rotated_template.py

```
@@ -103,11 +102,11 @@ class ArrowAngleMatcher:
     """
 
     def __init__(
-            self,
-            template_path: str | None = None,
-            template_center: Tuple[int, int] = None,
-            benchmark_width: int = 2560,
-            max_cache_scales: int = 12,
+        self,
+        template_path: str | None = None,
+        template_center: tuple[int, int] = None,
```

Ruff (0.16.2)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)